rosdep.yaml

```diff
@@ -1,280 +1,325 @@
+  ubuntu:
+    xenial: [ros-kinetic-behavior-tree-core]
+    bionic: [ros-melodic-behavior-tree-core]
+    focal: [ros-noetic-behavior-tree-core]
+behavior_tree_ros:
+  ubuntu:
+    xenial: [ros-kinetic-behavior-tree-ros]
+    bionic: [ros-melodic-behavior-tree-ros]
+    focal: [ros-noetic-behavior-tree-ros]
+behavior_tree_strawberry:
+  ubuntu:
+    xenial: [ros-kinetic-behavior-tree-strawberry]
+    bionic: [ros-melodic-behavior-tree-strawberry]
+    focal: [ros-noetic-behavior-tree-strawberry]
+
+# Sensors
+azure_kinect_ros_driver:
+  ubuntu:
+    xenial: [ros-kinetic-azure-kinect-ros-driver]
+    bionic: [ros-melodic-azure-kinect-ros-driver]
+    focal: [ros-noetic-azure-kinect-ros-driver]
+libk4abt1.1-dev:
+  ubuntu: [libk4abt1.1-dev]
+libk4abt1.1:
+  ubuntu: [libk4abt1.1]
+libk4a1.4-dev:
+  ubuntu: [libk4a1.4-dev]
+libk4a1.4:
+  ubuntu: [libk4a1.4]
+k4a-tools:
+  ubuntu: [k4a-tools]
+
+# TTS
+cerevoice_ros:
+  ubuntu:
+    xenial: [ros-kinetic-cerevoice-ros]
+    bionic: [ros-melodic-cerevoice-ros]
+    focal: [ros-noetic-cerevoice-ros]
+
+# Simulator & Visualization
+# This package has some useful tools for working with Actors in Gazebo simulations
+haru_description:
+  ubuntu:
+    xenial: [ros-kinetic-haru-description]
+    bionic: [ros-melodic-haru-description]
+    focal: [ros-noetic-haru-description]
+haru_gazebo:
+  ubuntu:
+    xenial: [ros-kinetic-haru-gazebo]
+    bionic: [ros-melodic-haru-gazebo]
+    focal: [ros-noetic-haru-gazebo]
+haru_simulator_examples:
+  ubuntu:
+    xenial: [ros-kinetic-haru-simulator-examples]
+    bionic: [ros-melodic-haru-simulator-examples]
+    focal: [ros-noetic-haru-simulator-examples]
+
+# Other dependencies not maintained by haru-project but required by Haru
+# ROS dependencies
+actors_simulator:
+  ubuntu:
+    xenial: [ros-kinetic-actors-simulator]
+    bionic: [ros-melodic-actors-simulator]
+    focal: [ros-noetic-actors-simulator]
+ros_tcp_endpoint:
+  ubuntu:
+    xenial: [ros-kinetic-ros-tcp-endpoint]
+    bionic: [ros-melodic-ros-tcp-endpoint]
+    focal: [ros-noetic-ros-tcp-endpoint]
+fkie_message_filters:
+  ubuntu:
+    xenial: [ros-kinetic-fkie-message-filters]
+    bionic: [ros-melodic-fkie-message-filters]
+    focal: [ros-noetic-fkie-message-filters]
+nix_common:
+  ubuntu:
+    xenial: [ros-kinetic-nix-common]
+    bionic: [ros-melodic-nix-common]
+    focal: [ros-noetic-nix-common]
 web_video_server:
   ubuntu:
+    xenial: [ros-kinetic-web-video-server]
     bionic: [ros-melodic-web-video-server]
     focal:   [ros-noetic-web-video-server]
-    xenial: [ros-kinetic-web-video-server]
-libssh:
+interactive_waypoint_markers:
   ubuntu:
-    bionic: [libssh-dev]
-    focal:  [libssh-dev]
-    xenial: [libssh-dev]
-
+    xenial: [ros-kinetic-interactive-waypoint-markers]
+    bionic: [ros-melodic-interactive-waypoint-markers]
+    focal: [ros-noetic-interactive-waypoint-markers]
 upo_markers:
   ubuntu:
+    xenial: [ros-kinetic-upo-markers]
     bionic: [ros-melodic-upo-markers]
```

Comment:
@lmercab do you know if  this package relevant to Haru Project?

Reply:
I thought this may be needed in the strawberry-ros-visualization repo (the package of RViz markers for Haru), but had a quick look and it does not seem to be the case. 

@jgimfig , could you check that out with @mattpeavy ?

Reply:
Any updates on this?

Reply:
@lmercab @lewkoo @jgimfig 

I have done some digging and it seems like those packages were added there in a pinch to get Debians for another project. Can be seen here:

https://github.com/haru-project/rosdistro/commit/b2975d50efeca56fcc6661b9ab3c18a7521efed5

They can be removed from this file.

Reply:
Removed.